tests/unit/lms/resources/lti_launch_test.py

```diff
@@ -158,6 +101,60 @@ def test_it_returns_the_js_config(self, pyramid_request, JSConfig):
         assert js_config == JSConfig.return_value
 
 
+class TestShouldUseSectionGroups:
+    @pytest.mark.parametrize(
+        "is_canvas,feature_flag,should_use_section_groups",
+        [
+            pytest.param(
+                True,
+                True,
+                True,
+                id="If we're in Canvas and the section_groups feature flag is "
+                "on then should_use_section_groups is True",
+            ),
+            pytest.param(
+                True,
+                False,
+                False,
+                id="If the section_groups feature flag is off then "
+                "should_use_section_groups is False",
+            ),
+            pytest.param(
+                False,
+                True,
+                False,
+                id="If we're not in Canvas then should_use_section_groups is " "False",
+            ),
+            pytest.param(
+                False,
+                True,
+                False,
+                id="If we're not in Canvas and the section_groups feature "
+                "flag is off then should_use_section_groups is False",
+            ),
+        ],
+    )
```

Comment:
Unfortunately you can't use keyword arguments with `pytest.param`, which would help a lot. That seems like the kind of thing that might be quite easy to do in pytest, might be successful as a feature request to pytest. For now I've rewritten it so that they each format onto one line as you suggested

Reply:
https://github.com/hypothesis/lms/pull/1680/commits/ff5647662013f052a5ea7a3fad269ae9f726929c